### PR TITLE
Use directly onecc instead of one-cmd

### LINKED ISSUE #318: Use `onecc` instead of `one-cmd`
Current ext uses `one-cmd` like `one-codegen`. These would be changed by `onecc`.

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's use directly onecc instead of one-cmd to resolve #318.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

**Comment by YongseopKim** [comment]
IHMO, I know this looks huge but this is inevitable because of change of `tool`.

**Comment by YongseopKim** [comment]
PTAL @Samsung/one-vscode

File: src/Project/Job.ts [hunk 1]
```diff
@@ -16,6 +16,18 @@
 
 import {ToolArgs} from './ToolArgs';
 
+// Currently `Job`s should be used by `onecc`
+// Ex1) JobCodegen
+// `onecc codegen arg0 arg1`
+// `onecc`: driver
+// `codegen`: tool
+// `arg0 arg1`: toolArgs
+// Ex1) JobImportTF
```
Comment (llFreetimell) [inline src/Project/Job.ts:25]:
```suggestion
// Ex2) JobImportTF
```

Comment (YongseopKim) [reply src/Project/Job.ts:25]: 👍

**Review by mhs4670go — APPROVED** [review]
LGTM

**Pull request merged**